Added natESM success story and updated references (#132)
DE-RSE/2023_paper-RSE-groups: paper.tex

\addbibresource{positionpaper.bib}
\setcounter{biburllcpenalty}{100}

> % Um die Auffindbarkeit des Papers zu erhöhen, schlage ich vor, den Titel zu aendern, denn nicht alle kennen den Begriff RSE, der zudem auch noch unterschiedlich gelesen werden kann. Dies wird die Auffindbarkeit des Papers erhöhen.

Reviewer: ```suggestion
```

\addbibresource{positionpaper.bib}
\setcounter{biburllcpenalty}{100}

> %\title{Establishing central RSE units in German research institutions} 

Reviewer: ```suggestion

```

\maketitle

\abstract{%
This position paper urges decision makers in Germany to establish central Research Software Engineering (RSE) units within their institutions.
Focus is not put primarily on the establishment of RSE services in general, as this has been done already elsewhere.
Instead, we highlight central RSE units.
Motivations for their existance are discussed, underpinned by working examples both in neighbouring fields as well as outside of Germany.

> The heart of this paper is a vision of a central RSE unit, it's structure, and the definition of nine core support modules such a unit may provide.

Reviewer 2: ```suggestion
The heart of this paper is a vision of a central RSE unit, its structure, and the definition of nine core support modules such a unit may provide.
```

while reviewing I have found this typo

\label{sec:vision}

In this section, we describe our vision for central RSE units at research institutions in Germany.

> The term institutions is, for our purposes, a broad term that covers universities, other colleges, associations like Max-Planck, Helmholtz, Fraunhofer, or Leibniz, as well as other research performing organisations.

Reviewer 2: ```suggestion
The term institution is, for our purposes, a broad term that includes universities, other colleges, associations like Max-Planck, Helmholtz, Fraunhofer, or Leibniz, as well as other research performing organisations.
```
I think _include_ is the right word here although cover would work as well

The RSE unit thus contributes to the RSE communities on a regional, national or international level on the one hand and opens these up to the local RSEs and enables networking on the other hand.
It organises the bidirectional exchange between the local and the global community and is the central hub for information coming both ways.

\begin{mdframed}
\textbf{Success Story:} As a national initiative integrating Research Software Engineering (RSE) principles, natESM (the national Earth System Modeling strategy~\autocite{Ehlert2025}) is developing a sustainable and high-performance Earth system modeling capability for Germany while addressing several of the key aspects outlined above. By coordinating RSE activities across multiple institutions through a well-structured sprint process, natESM fosters collaboration between scientists and RSEs, ensuring that future Earth system models remain efficient, maintainable, and HPC-compatible. Within this process, natESM also provides structured consultation services, supporting researchers in adopting modern software development practices and strengthening the technical foundations of their models.

> Beyond development and maintenance, regular training events and workshops equip scientists with the expertise needed to work effectively within the natESM system. A GitLab structure, community newsletters, a Mattermost channel for direct exchange, and a dedicated website ensure transparency and engagement across the national Earth system modeling community. Through this integrated approach, natESM reinforces the technical backbone of Earth system and climate modeling, ensuring that it remains adaptable, scalable, and future-proof.

Reviewer 2: ```suggestion
Beyond development and maintenance, regular training events and workshops equip scientists with the expertise needed to work effectively within the natESM system. A GitLab service, community newsletters, a Mattermost channel for direct exchange, and a dedicated website ensure transparency and engagement across the national Earth system modeling community. Through this integrated approach, natESM reinforces the technical backbone of Earth system and climate modeling, ensuring that it remains adaptable, scalable, and future-proof.
```

Author: Good catch :). Thanks for your help!

The RSE unit thus contributes to the RSE communities on a regional, national or international level on the one hand and opens these up to the local RSEs and enables networking on the other hand.
It organises the bidirectional exchange between the local and the global community and is the central hub for information coming both ways.

\begin{mdframed}

> \textbf{Success Story:} As a national initiative integrating Research Software Engineering (RSE) principles, natESM (the national Earth System Modeling strategy~\autocite{Ehlert2025}) is developing a sustainable and high-performance Earth system modeling capability for Germany while addressing several of the key aspects outlined above. By coordinating RSE activities across multiple institutions through a well-structured sprint process, natESM fosters collaboration between scientists and RSEs, ensuring that future Earth system models remain efficient, maintainable, and HPC-compatible. Within this process, natESM also provides structured consultation services, supporting researchers in adopting modern software development practices and strengthening the technical foundations of their models.

Reviewer: ```suggestion
\textbf{Success Story:} As a national initiative integrating Research Software Engineering (RSE) principles, natESM (the national Earth System Modeling strategy~\autocite{Ehlert2025}) is developing a sustainable and high-performance Earth system modeling capability for Germany while addressing several of the key aspects outlined above.
By coordinating RSE activities across multiple institutions through a well-structured sprint process, natESM fosters collaboration between scientists and RSEs, ensuring that future Earth system models remain efficient, maintainable, and HPC-compatible.
Within this process, natESM also provides structured consultation services, supporting researchers in adopting modern software development practices and strengthening the technical foundations of their models.
```

Reviewer: semantic line breaks

The RSE unit thus contributes to the RSE communities on a regional, national or international level on the one hand and opens these up to the local RSEs and enables networking on the other hand.
It organises the bidirectional exchange between the local and the global community and is the central hub for information coming both ways.

\begin{mdframed}
\textbf{Success Story:} As a national initiative integrating Research Software Engineering (RSE) principles, natESM (the national Earth System Modeling strategy~\autocite{Ehlert2025}) is developing a sustainable and high-performance Earth system modeling capability for Germany while addressing several of the key aspects outlined above.
By coordinating RSE activities across multiple institutions through a well-structured sprint process, natESM fosters collaboration between scientists and RSEs, ensuring that future Earth system models remain efficient, maintainable, and HPC-compatible.
Within this process, natESM also provides structured consultation services, supporting researchers in adopting modern software development practices and strengthening the technical foundations of their models.

> Beyond development and maintenance, regular training events and workshops equip scientists with the expertise needed to work effectively within the natESM system. A GitLab service, community newsletters, a Mattermost channel for direct exchange, and a dedicated website ensure transparency and engagement across the national Earth system modeling community. Through this integrated approach, natESM reinforces the technical backbone of Earth system and climate modeling, ensuring that it remains adaptable, scalable, and future-proof.

Reviewer: ```suggestion
Beyond development and maintenance, regular training events and workshops equip scientists with the expertise needed to work effectively within the natESM system. A GitLab service, community newsletters, a Mattermost channel for direct exchange, and a dedicated website ensure transparency and engagement across the national Earth system modeling community.
Through this integrated approach, natESM reinforces the technical backbone of Earth system and climate modeling, ensuring that it remains adaptable, scalable, and future-proof.
```

The RSE unit thus contributes to the RSE communities on a regional, national or international level on the one hand and opens these up to the local RSEs and enables networking on the other hand.
It organises the bidirectional exchange between the local and the global community and is the central hub for information coming both ways.

\begin{mdframed}

> \textbf{Success Story:} As a national initiative integrating Research Software Engineering (RSE) principles, natESM (the national Earth System Modeling strategy~\autocite{Ehlert2025}) is developing a sustainable and high-performance Earth system modeling capability for Germany while addressing several of the key aspects outlined above.

Reviewer: ```suggestion
\textbf{Success Story:} As a national initiative integrating Research Software Engineering (RSE) principles, natESM (the national Earth System Modelling strategy~\autocite{Ehlert2025}) is developing a sustainable and high-performance Earth system modelling capability for Germany while addressing several of the key aspects outlined above.
```

By coordinating RSE activities across multiple institutions through a well-structured sprint process, natESM fosters collaboration between scientists and RSEs, ensuring that future Earth system models remain efficient, maintainable, and HPC-compatible.
Within this process, natESM also provides structured consultation services, supporting researchers in adopting modern software development practices and strengthening the technical foundations of their models.

> Beyond development and maintenance, regular training events and workshops equip scientists with the expertise needed to work effectively within the natESM system. A GitLab service, community newsletters, a Mattermost channel for direct exchange, and a dedicated website ensure transparency and engagement across the national Earth system modeling community.

Reviewer: ```suggestion
Beyond development and maintenance, regular training events and workshops equip scientists with the expertise needed to work effectively within the natESM system. A GitLab service, community newsletters, a Mattermost channel for direct exchange, and a dedicated website ensure transparency and engagement across the national Earth system modelling community.
```

> Through this integrated approach, natESM reinforces the technical backbone of Earth system and climate modeling, ensuring that it remains adaptable, scalable, and future-proof.

Reviewer: ```suggestion
Through this integrated approach, natESM reinforces the technical backbone of Earth system and climate modelling, ensuring that it remains adaptable, scalable, and future-proof.
```
